src/entrypoints/variables.ts

```diff
@@ -1,11 +1,15 @@
-export const NODE_OPTIONS = [
+import { v4 as uuidv4 } from "uuid";
+export const NODE_OPTIONS: CustomStyleNode[] = [
   { label: "Paragraph", value: "paragraph" },
   { label: "Heading", value: "heading" },
-] as const;
+];
 
-export const DUMMY_CUSTOM_STYLE: CustomStyle = {
-  title: "Dummy Style",
-  css: "color: darkgrey;\nfont-size: 20px;\nfont-style: italic;",
-  node: NODE_OPTIONS[0],
-  isOpen: true,
+export const DUMMY_CUSTOM_STYLE = () => {
+  return {
+    id: uuidv4(),
```

Comment:
Why is this a uuid? I would imagine this to be the `slug` (in combination with the node type to allow for multiple entries) based on the title. We want that slug to be available in the node when rendering it on the visitor-facing website